### PR TITLE
Added Blake2b-256

### PR DESCRIPTION
**Pull request by swanandx**

### CODE REVIEW

**Comment by bee-san** [comment]
CI isn't running here so I can't tell if the tests work lol

File: tests/test_main.py [hunk 1]
```diff
@@ -193,4 +192,10 @@ def test_androidbackup():
     hashes = ["$ab$5*0*10000*b8900e4885ff9cad8f01ee1957a43bd633fea12491440514ae27aa83f2f5c006ec7e7fa0bce040add619919b4eb60608304b7d571a2ed87fd58c9ad6bc5fcf4c*7d254d93e16be9312fb1ccbfc6265c40cb0c5eab7b605a95a116e2383fb1cf12b688223f96221dcd2bf5410d4ca6f90e0789ee00157fa91658b42665d6b6844c*fc9f6be604d1c59ac32664ec2c5b9b30*00c4972149af3adcc235899e9d20611ea6e8de2212afcb9fcfefde7e35b691c2d0994eb47e4f9a260526ba47f4caea71af9c7fadcd5685d50126276f6acdd59966528b13ccc26036a0eaba2f2451aa64b05766d0edd03c988dcf87e2a9eec52d"]
 
     x = runner.api_return_hashes_as_json(hashes)
-    assert "Android Backup" in x
\ No newline at end of file
+    assert "Android Backup" in x
+
+def test_balke2b256():
```
Comment (SkeletalDemise) [inline tests/test_main.py:197]: There's a spelling mistake. Change it to `test_blake2b256`.

Comment (swanandx) [reply tests/test_main.py]: Fixed it.

**Review by SkeletalDemise — APPROVED** [review]

**Comment by SkeletalDemise** [comment]
@all-contributors please add @swanandx for code

**Pull request merged**